server/rest/src/test/java/org/infinispan/rest/resources/CacheManagerResourceTest.java

```diff
@@ -142,7 +142,8 @@ public void testCaches() throws Exception {
       String json = response.getContentAsString();
       JsonNode jsonNode = mapper.readTree(json);
       List<String> names = asText(jsonNode.findValues("name"));
-      Set<String> expectedNames = Util.asSet("defaultcache", "___protobuf_metadata", "___script_cache", "cache1", "cache2");
+      Set<String> expectedNames = Util.asSet("defaultcache", "cache1", "cache2");
```

Comment:
As commented before, we should not change the API return payloads, but add a new request parameter to filter out the internal caches

Reply:
since this endpoint has been created for the console needs and documented as experimental, can't we change it not to include internals but only if I add a filter to ask for them? in any case, we do have the option to get the internal cache and the details, since the cache manager api does not filter internal cache names

Reply:
I mean, the defined caches properties